bitfield/admin.py

```diff
@@ -1,7 +1,11 @@
+import django
 import six
 
 from django.core.exceptions import ValidationError
-from django.utils.translation import ugettext_lazy as _
+if django.VERSION < (2, 0):
```

Comment:
I think you want 3.0 here; see https://docs.djangoproject.com/en/3.2/releases/3.0/#id3.

Reply:
It was aliased starting in 2.0, deprecated in 3.0. So either should work, but I'd be fine with 3.0 to be safer.